daemon/api_general.go

```diff
@@ -59,14 +59,14 @@ var (
 		Path:        "/v2/changes/{id}",
 		GET:         getChange,
 		POST:        abortChange,
-		ReadAccess:  openAccess{},
+		ReadAccess:  interfaceOpenAccess{Interface: "snap-refresh-observe"},
```

Comment:
given that we are applying these checkers now to very broad endpoints we should add a TODO to interfaceOpenAccess and interfaceAuthenticatedAccess to take a list of interfaces and not just one. We don't need to do that now though

Reply:
Yes, I can add this support in a followup PR